util/_dump-redis.sh

```
@@ -1,7 +1,52 @@
+    HOST_VAR="REDIS_CACHE_HOST"
+    PORT_VAR="REDIS_CACHE_PORT"
+    PASS_VAR="REDIS_CACHE_PASS"
+    ;;
+  assessment)
+    HOST_VAR="REDIS_ASSESSMENT_HOST"
+    PORT_VAR="REDIS_ASSESSMENT_PORT"
+    PASS_VAR="REDIS_ASSESSMENT_PASS"
+    ;;
+esac
+
+infisical run --env $ENV -- env HOST_VAR="$HOST_VAR" PORT_VAR="$PORT_VAR" PASS_VAR="$PASS_VAR" \ 
```

Shellcheck (0.11.0)

[error] Delete trailing spaces after \ to break line (or use quotes for literal space).

(SC1101)